source/configuration/program-detail-page.md

```diff
@@ -168,15 +168,15 @@ const leftPanel = {
 
 ##### External Links to Labels and Values
 1. External links shall start with 'http://' or'https://'
-2. External links shall show-up with 'externalLinkIcon' 
+2. External links shall show-up with 'externalLinkIcon'
 3. External link shall be opened in a new tab.
 4. Dynamic links can be generated by passing a valid table filed to '{}'. For example, 'https://pubmed.ncbi.nlm.nih.gov/{pubmed_id}' shall link to 'https://pubmed.ncbi.nlm.nih.gov/29860917/'
 
 ## Configuring the Right Hand Panel
 In the current version of Bento, you can (a) add a donut widget that displays the counts by value of a key data entity in your data sharing platform and (b) add a count of files stored for the given program.
```

Comment:
Replace with..."In the current version of Bento donut widgets can be added to display the counts by value of any key data entity in the data sharing platform and (b) adding a count of files stored for a given program."